release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -287,7 +287,8 @@ In {product-title} 4.7, the Insights Operator collects the following additional
 * The `ContainerRuntimeConfig` and `MachineConfigPools` configuration files to verify container storage limits
 * The configuration files for all available `operator.openshift.io` control pane resources to identify Operators in unmanaged states
 * The `NetNamespaces` names, including their `netID` and egress IP addresses
-* The `StatefulSet` configuration definitions from the {product-title} default namespaces to check the Prometheus monitoring storage type
+* A list of all installed OLM Operators, including version information
+* The Persistent Volume definition, if used in the Image Registry Operator configuration
```

Comment:
Does "Persistent Volume" need some markup?
Same question for "Image Registry"?

Reply:
I don't think it's appropriate for Persistent Volume, but I changed Image Registry to `openshift-image-registry` namespace